indico/modules/events/timetable/templates/move_entry.html

```diff
@@ -1,75 +1,152 @@
+                        <span class="title"><strong>{% trans %}Top level timetable{% endtrans %}</strong></span>
                         <span class="start-dt">
-                            ({{ tt_entry.start_dt|format_time(timezone=event.timezone) }} -
-                             {{ tt_entry.end_dt|format_time(timezone=event.timezone) }})
+                            {%- if parent_entry is none and day == current_day %}
+                                {%- trans %}Currently at this level{% endtrans %}
+                            {% endif -%}
+                        </span>
+                        <span class="move-btn">
+                            <button class="i-button highlight
+                                           {%- if parent_entry is none and day == current_day %} active{% endif %}">
+                                {% trans %}Move here{% endtrans %}
+                            </button>
                         </span>
-                        <span class="color" style="background: #{{ block.session.colors.background }}"></span>
                     </li>
-                {% endfor %}
-            </ul>
-        {% endfor %}
+                    {% for tt_entry in tt_entries %}
+                        {% set block = tt_entry.session_block %}
+                        <li class="item block {%- if tt_entry == parent_entry %} selected{% endif %}"
+                            data-entry-id="{{ tt_entry.id }}"
+                            data-item-id="{{ tt_entry.id }}"
+                            data-title="{% trans %}Move entry{% endtrans %}">
+                            <span class="item-icon color"
+                                  style="background: #{{ block.session.colors.background }}"></span>
+                            <span class="title">{{ block.full_title }}</span>
+                            <span class="start-dt">
+                                {%- if tt_entry == parent_entry %}
+                                    ({%- trans %}Currently at this level{% endtrans %})
+                                {% endif -%}
+                                {{ tt_entry.start_dt | format_time(timezone=event.timezone) }} -
+                                {{ tt_entry.end_dt | format_time(timezone=event.timezone) }}
+                            </span>
+                            <span class="move-btn">
+                                <button class="i-button highlight {%- if tt_entry == parent_entry %} active{% endif %}">
+                                    {% trans %}Move here{% endtrans %}
+                                </button>
+                            </span>
+                        </li>
+                    {% endfor %}
+                </ul>
+            {% endfor %}
+        </div>
     </div>
-
-    <input id="move-entry-data" type="hidden">
 </div>
 
 <script>
-    $('.days [data-day]').on('click', function() {
-        var $this = $(this);
+    (function(global) {
```

Comment:
i'd remove the argument since it's not being used (also the `window` at the end)